ArcGISRuntimeSDKQt_CppSamples/AR/DisplayScenesInTabletopAR/DisplayScenesInTabletopAR.h

```diff
@@ -64,6 +65,7 @@ private slots:
   Esri::ArcGISRuntime::SceneQuickView* m_sceneView = nullptr;
   Esri::ArcGISRuntime::MobileScenePackage* m_scenePackage = nullptr;
   Esri::ArcGISRuntime::Camera m_originCamera;
+  PermissionsHelper* m_permissionsHelper;
```

Comment:
Sorry I missed this in the previous review.
```suggestion
  PermissionsHelper* m_permissionsHelper = nullptr;
```